go.mod

```diff
@@ -12,10 +12,10 @@ require (
 	github.com/kong/go-slugify v1.0.0
 	github.com/onsi/ginkgo/v2 v2.22.2
 	github.com/onsi/gomega v1.36.2
-	github.com/pb33f/libopenapi v0.16.13
+	github.com/pb33f/libopenapi v0.24.0
```

Comment:
@mheap this is not necessary for speakeasy jsonpath, correct? 
This breaks vacuum on deck, i'll downgrade it to 0.23.0 (which is required by vacuum), and bump vacuum version in deck together.